prosopo.config.ts

```diff
@@ -28,12 +28,19 @@ export default {
             }
         }
     },
+    captchas: {
```

Comment:
This was meant to be a config option where the user could select either choose between serving `solved` **only** or `solved` and `unsolved`.

`captchas : { solvedState: [solved Or solvedAndUnsolved], number: n }`

Reply:
Either that or you do what you were doing but make it more explicit

`captchas : { solved: true, unsolved: true, number: n }`

Then just split the number of captchas equally if both `unsolved` and `solved` are set to `true`.

You would also need to introduce an error if the user put something like this:

`captchas : { solved: false, unsolved: true, number: n }`

because this wouldn't make any sense. So maybe the option in the previous comment is better.

Reply:
In your config

    `captchas: { solved: { count: 1 }, solvedAndUnsolved: { count: 1 }},`

If someone puts

`solved = 2` and `solvedAndUnsolved = 2`

how many solved and how many unsolved captchas would the user expect to receive? This isn't clear.

You are overriding the config value `solved` simply by the `solvedAndUnsolvedCount` value existing. This needs to be more explicit.